src/Containers/PromotionContainer.php

```diff
@@ -12,6 +13,7 @@
  */
 class PromotionContainer
 {
+    use Loggable;
```

Comment:
Please check all the usages of this Trait. I guess this was left over from debugging?